fastai/model.py

```diff
@@ -116,6 +116,7 @@ def fit(model, data, epochs, opt, crit, metrics=None, callbacks=None, stepper=St
         stepper.reset(True)
         t = tqdm(iter(data.trn_dl), leave=False, total=num_batch)
         i = 0
+        if all_val: val_iter = iter_batch(data.val_dl)
```

Comment:
Why isn't this a python iterator? BTW since it's a class I'd expect to see CamelCase